configmodels/config_msg.go

```diff
@@ -22,14 +22,14 @@ const (
 )
 
 type ConfigMessage struct {
-	MsgType      int
-	MsgMethod    int
 	DevGroup     *DeviceGroups
 	Slice        *Slice
 	DevGroupName string
 	SliceName    string
 	AuthSubData  *models.AuthenticationSubscription
```

Comment:
Should not this also be changed?
```suggestion
	AuthSubData  *models.AuthenticationSubscription
	DevGroupName string
	SliceName    string
```

Reply:
I don't get any error when running `golangci-lint run` 
Also from what I have seen pointers and string take the same place so there would not be any change on the total size of the struct 
But I agree it would look cleaner